packages/eas-cli/src/utils/files.ts

```diff
@@ -26,3 +26,32 @@ export async function maybeRenameExistingFileAsync(projectDir: string, filename:
     await fs.rename(desiredFilePath, path.resolve(projectDir, getRenamedFilename(filename, num)));
   }
 }
+
+export function formatBytes(bytes: number): string {
```

Comment:
I'd prefer to use an npm package for that. E.g. https://github.com/optoolco/nice-bytes

Reply:
discussed on slack